modules/admin_manual/pages/configuration/user/user_roles.adoc

```diff
@@ -11,10 +11,9 @@ high-level overview of each type.
 
 * Is not a regular user.
 * Has access to specific content made available via public links.
-** Can be password-protected +
+** Can be password-protected
 (optional, enforced, policy-enforced).
-+
-- Can have an expiration date +
+** Can have an expiration date +
```

Comment:
Can you please remove the trailing `+`?

Reply:
tbh, no mathew,
you are just looking to a trailing `+` but not to the result. pls see the screenshot below.
all items here are rendered like that scheme and this is a big improvement for readabilitry.
![image](https://user-images.githubusercontent.com/3321281/57621749-8cd2b080-758c-11e9-9530-54f2a9dfc706.png)

Reply:
Actually, I'm asking for the `+` to be removed, as I don't believe that the reflow is natural nor aids comprehension. Additionally, this needs to be properly explored and discussed, as it has implications for the remainder of the documentation.